Formula/trec_eval.rb

```diff
@@ -0,0 +1,34 @@
+class TrecEval < Formula
+  desc "Evaluation software used in the Text Retrieval Conference"
+  homepage "https://trec.nist.gov/"
+  url "https://github.com/usnistgov/trec_eval/archive/v9.0.8.tar.gz"
+  sha256 "c3994a73103ec842e12df693749584a45814c35c36dcc15f38984bd463566ba1"
+  license :public_domain
+
+  def install
+    system "make"
+    bin.install "trec_eval"
+  end
+
+  test do
+    qrels = <<~EOS
+      301 0 q1 0
+      302 0 q2 1
+    EOS
+    results = <<~EOS
+      301	Q0 q1 3	1.23 testid
+      302	Q0 q2 50 2.34 testid
+    EOS
+    out = "runid                 \tall\ttestid\n\
+num_q                 \tall\t2\n\
+map                   \tall\t0.5000\n\
+P_10                  \tall\t0.0500\n\
+recall_10             \tall\t0.5000\n\
+ndcg_cut_10           \tall\t0.5000\n"
```

Comment:
```suggestion
    out = <<~EOS
    runid                 \tall\ttestid
    num_q                 \tall\t2
    map                   \tall\t0.5000
    P_10                  \tall\t0.0500
    recall_10             \tall\t0.5000
    ndcg_cut_10           \tall\t0.5000
    EOS
```

Reply:
Thanks! Not quite familiar with ruby strings yet

Reply:
meh, the auditor doesn't like that :/

Reply:
Suggest always running `brew audit --strict --online --git --fix trec_eval` before pushing. That should catch (and fix if it can) trivial syntax issues, so you don't have to keep round-tripping.

Reply:
Thank you, didn't know about the `--fix` flag! Now it should be good